taurus/entity/bounds/composition_bounds.py

```diff
@@ -41,30 +39,6 @@ def components(self, value):
         if not all(isinstance(x, str) for x in self.components):
             raise ValueError("All the components must be strings")
 
-    def validate(self, value: BaseValue) -> bool:
-        """
-        Check if a value has all of the required components.
-
-        Parameters
-        ----------
-        value: BaseValue
-            Value to validate. In order to be valid, must be a
-            :py:class:`CompositionValue <taurus.entity.value.composition_value.CompositionValue>`
-            and have all of the required components.
-
-        Returns
-        -------
-        bool
-            True if the value is a composition with all required components.
-
-        """
-        if not super().validate(value):
-            return False
-        if not isinstance(value, CompositionValue):
-            return False
-
-        return all(x in self.components for x in value.components)
-
```

Comment:
I disagree with removing this

Reply:
If we don't have validate on all bounds, then we shouldn't have it on any.

Reply:
I think we should validate all values against bounds. I only support removing validation when it touches templates or other resources that have an external life someplace else.